packages/strapi-helper-plugin/lib/src/components/NavLink/styles.scss

```diff
@@ -0,0 +1,28 @@
+a.navLink {
```

Comment:
We should define and follow a guideline for the order of the CSS properties. For example, I like to follow this one but we can change it if you prefer another one.

- Alignment properties (display, justify-items, top, left)
- Block properties (width, height, margin, padding)
- Content properties (font, colors)
- Animation and extra styles

```
.myClass{
  display: flex;
  justify-items: center;
  width: 100%;
  height: 50px;
  padding: 0 50 25px;
  margin: 0;
  font-size: 2rem;
  font-weight: 600;
  color: #f30;
  transition: color .25s ease-out;
}
```

Why should we do this? If we follow guidelines like this one, it repeats a pattern and it's easier to debug, to read and to maintain. The impact will be huge in the long-term.